src/Chirp.Razor/CheepService.cs

```diff
@@ -51,7 +51,13 @@ public async Task Create(CheepCreateDTO cheep)
     }
     public async Task AddAuthor(string name, string email)
     {
-        await _authorRepository.AddAuthor(name, email);
+        _authorRepository.AddAuthor(name, email);
+    }   
+    public void DeleteCheepsFromAuthor(int authorid){
+        _cheepRepository.DeleteCheepsFromAuthor(authorid);
+    }
+    public void DeleteAuthor(int authorId){
```

Comment:
We should change this method to be asynchronous.

I am happy to help if necessary.

Reply:
Here I am reffereing to the DeleteAuthor method